ui/hooks/useTransactionDisplayData.js

```diff
@@ -383,8 +388,14 @@ export function useTransactionDisplayData(transactionGroup) {
   } else if (type === TransactionType.bridge) {
     title = t('bridgeToChain', [destChainName || '']);
     category = TransactionGroupCategory.bridge;
-    // TODO add primaryDisplayValue,primarySuffix
-    // also secondaryDisplayValue, secondarySuffix
+    // TODO also secondaryDisplayValue, secondarySuffix
+
+    primarySuffix = primaryTransaction.sourceTokenSymbol;
+    primaryDisplayValue = calcTokenAmount(
+      primaryTransaction.sourceTokenAmount,
```

Comment:
Does sourceTokenAmount include the MM fees?

Reply:
`sourceTokenAmount` comes from `quoteResponse.quote.srcTokenAmount`, so it does not include MM fees. What should the activity list item display? The amount that has the fees accounted for or the actual total amount that left your wallet?

Reply:
It should be the amount including fees, which is equal to the amount that left the user's wallet

Reply:
the quote's `srcTokenAmount` field only reflects the amount bridged after the MetaMask fees are taken. so `srcTokenAmount + (0.875% * srcTokenAmount)` always leaves the user's wallet

Reply:
Are you sure about the `quote.srcTokenAmount`? I'm looking at a quote for 0.01 UNI and it's showing `
srcTokenAmount: "10000000000000000"`

Reply:
Done in [02c54fb](https://github.com/MetaMask/metamask-extension/pull/28899/commits/02c54fbf1a9232d0b58f161b11d18691f61b1f6c)